megazords/ios-rust/MozillaTestServices/MozillaTestServices.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,16 +1,14 @@
 {
-  "object": {
```

Comment:
QQ: Why are these changes necessary?

Reply:
heh snuck in here, I imagine something to do with Xcode running and me switching branches 🤷 got rid of it!